scene/gui/range.cpp

```diff
@@ -183,6 +183,8 @@ double Range::get_as_ratio() const {
 		return CLAMP((v - exp_min) / (exp_max - exp_min), 0, 1);
 
 	} else {
+		if (Math::is_zero_approx(get_max() - get_min()))
```

Comment:
You can use `is_equal_approx(get_max(), get_min())` instead.

Reply:
There's the same problem in the previous branch a priori. IMO it would be worth simply throwing an error if min == max.